deps.windows/40-detours.ps1

```diff
@@ -2,26 +2,26 @@ param(
     [string] $Name = 'detours',
     [string] $Version = '4.0.1',
     [string] $Uri = 'https://github.com/microsoft/detours.git',
-    [string] $Hash = 'e4bfd6b03e50de46b47abfbd1e46b384f0c5f833'
+    [string] $Hash = '734ac64899c44933151c1335f6ef54a590219221'
```

Comment:
Is there a specific change needed between these hashes? Last time I asked about updating this, I was asked/told not to update from the stable tag, old as it may be.

Reply:
Only the most recent tags support ARM64 at all in my tests.

Reply:
FWIW, we _are_ using the most recent tag, but there have been new commits since then.

Reply:
Until a new stable tag emerges, we're going to continue using the stable tag for x64/x86, but a recent commit for arm64.